documentation/bibliography.bib

```diff
@@ -1369,73 +1394,102 @@ @incollection{Halley1809
+  venue     = {Minneapolis, Minnesota},
 }
 
 @inproceedings{JonesAnderson2012,
-  author    = {Jones, Brandon A. and Anderson, Rodney L.},
-  booktitle = {AAS/AIAA Spaceflight Mechanics Meeting},
-  date      = {2012},
-  series    = {Advances in the Astronautical Sciences},
-  title     = {A Survey of Symplectic and Collocation Integration Methods for Orbit Propagation},
-  volume    = {143},
+  author       = {Jones, Brandon A. and Anderson, Rodney L.},
+  eventdate    = {2012-01-29/2012-02-02},
+  eprint       = {http://hdl.handle.net/2014/42453},
+  booktitle    = {22nd AAS/AIAA Spaceflight Mechanics Meeting},
+  date         = {2012},
+  note         = {AAS 12-214},
+  organization = {American Astronautical Society},
+  publisher    = {Univelt},
+  series       = {Advances in the Astronautical Sciences},
+  title        = {A Survey of Symplectic and Collocation Integration Methods for Orbit Propagation},
+  venue        = {Charleston, South Carolina},
+  number       = {143},
 }
 
 @inproceedings{KrishnamoorthyMenon2013,
   author    = {Krishnamoorthy, A. and Menon, D.},
+  eventdate = {2013-09-26/2013-09-28},
+  publisher = {IEEE},
   booktitle = {Signal Processing: Algorithms, Architectures, Arrangements, and Applications},
   date      = {2013},
   pages     = {70--72},
   title     = {Matrix Inversion Using Cholesky Decomposition},
+  venue     = {Poznań, Poland},
 }
 
 @inproceedings{PellegriniRussel2014,
-  author    = {Pellegrini, Etienne and Russell, Ryan P.},
-  booktitle = {AAS/AIAA Spaceflight Mechanics Meeting},
-  date      = {2014},
-  series    = {Advances in the Astronautical Sciences},
-  title     = {F and G Taylor Series Solutions to the Circular Restricted Three Body Problem},
-  volume    = {152},
+  author       = {Pellegrini, Etienne and Russell, Ryan P.},
+  eventdate    = {2014-01-26/2014-01-30},
+  booktitle    = {24th AAS/AIAA Spaceflight Mechanics Meeting},
+  date         = {2014},
+  note         = {AAS 14-237},
+  organization = {American Astronautical Society},
+  publisher    = {Univelt},
+  series       = {Advances in the Astronautical Sciences},
+  title        = {F and G Taylor Series Solutions to the Circular Restricted Three Body Problem},
+  venue        = {Santa Fe, New Mexico},
+  number       = {152},
 }
 
 @inproceedings{RussellLara2006,
   author    = {Russell, R. P. and Lara, M.},
-  booktitle = {AIAA/AAS Astrodynamics Specialist Conference},
-  date      = {2006-08},
+  booktitle = {AIAA/AAS Astrodynamics Specialist Conference and Exhibit},
+  eventdate = {2006-08-21/2006-08-24},
+  date      = {2006-08-24T09:00:00},
   doi       = {10.2514/6.2006-6750},
-  series    = {Collection of Technical Papers},
+  note      = {AIAA 2006-6750},
+  publisher = {American Institute of Aeronautics and Astronautics},
+  series    = {Guidance, Navigation, and Control and Co-located Conferences},
   title     = {Repeat Ground Track Lunar Orbits in the Full-Potential Plus Third-Body Problem},
-  volume    = {3},
```

Comment:
Why did we lose the volume?

Reply:
As best I can tell it was nonsense.
_Guidance, Navigation, and Control and Co-located Conferences_ is unnumbered, so it was not a `number` (a lot of `volumes` were « volume within the `series` » which should go into `number` in `@inproceedings`).
_AIAA/AAS Astrodynamics Specialist Conference and Exhibit_ (2006) does not appear to be partitioned in `volume`s either: https://arc.aiaa.org/doi/book/10.2514/MAST06.

Note that the previous value of `series`, _Collection of Technical Papers_, seems nonsensical.

Reply:
This came from [the citation on ResearchGate](https://www.researchgate.net/publication/228830968_Repeat_Ground_Track_Lunar_Orbits_in_the_Full-Potential_Plus_Third-Body_Problem/citation/download).

Reply:
Ah, it looks like in print these proceedings appeared in the three-volume ISBN 1563478226, _A collection of technical papers_:
1. https://www.tib.eu/en/search/id/TIBKAT:521489660/A-collection-of-technical-papers-AIAA-AAS-Astrodynamics?cHash=6349bf79113ed67885ab83701e652639;
2. https://www.tib.eu/en/search/id/TIBKAT:521489873/A-collection-of-technical-papers-AIAA-AAS-Astrodynamics?cHash=cd433c128bcf43589e005c9f781fdb37;
3. https://www.tib.eu/en/search/id/TIBKAT:521489903/A-collection-of-technical-papers-AIAA-AAS-Astrodynamics?cHash=cb1d3358a90d8b8658ca3bb358e99946.

See https://www.tib.eu/en/search/id/BLCP:CN065746138/Repeat-Ground-Track-Lunar-Orbits-in-the-Full-Potential?cHash=f086b9f29f698e135b8d8d7aa257c363.

This print version has page numbers 1935–1954 in that citation, whereas the ARC PDF is numbered 1–20…

If we want to cite the print book we should do the same for the other paper from the AIAA/AAS Astrodynamics Specialist Conference, [Jon12], in https://www.tib.eu/en/search/id/TIBKAT:769080677/AIAA-AAS-Astrodynamics-Specialist-Conference-2012?cHash=190c9597dcb0a79c120396334a3fb587—note that for that one the POD publisher is [Curran](http://www.proceedings.com/15920.html), and the AIAA would need to move from `publisher` to `organization`.

Reply:
Added the print metadata to both entries.